src/app/molecules/message/Message.jsx

```diff
@@ -734,7 +735,7 @@ function Message({
         {roomTimeline && !isEditing && (
           <MessageOptions
             roomTimeline={roomTimeline}
-            mEvent={mEvent}
+            mEvent={editedMEvent !== undefined ? editedMEvent : mEvent}
```

Comment:
changing this mEvent is not a good idea as other option in menu also depend on this. If we want to get the edited message we can get it from `roomTimeline.editedTimeline` inside MessageOptions

Reply:
I've moved the changes to `MessageOptions`. Please review the changes.